Code/Framework/AzCore/AzCore/IO/Streamer/RequestPath.h

```diff
@@ -29,7 +29,7 @@ namespace AZ
             RequestPath(RequestPath&& rhs) = default;
             // Allow path views to be cast to a request path.
             RequestPath(AZ::IO::PathView path);
-
+            
```

Comment:
nitpick: extra whitespace added.

FYI: if you use Visual Studio Code or Visual Studio, you can have it delete trailing whitespace on save.

Reply:
Removed in the next update.
The option to delete trailing whitespace doesn't seem to be available in Visual Studio, just Visual Studio Code.